### DISCUSSION TITLE
User account blocked on sign-up

### DISCUSSION THREAD

**Discussion by TCU645** [vercel/community#3321] [Help] [answered]
### Summary

I was trying to set up a GitHub account and a Vercel account for community work we are doing in college, however, when I tried to create the Vercel account it immediately said that this user account was blocked. It was a fresh Email that has never been used, the GitHub account only has the webpage repo, which is the one we are supposed to publish.
I don't know what to do, I tried to contact support, but you need to log in and when trying to I get the error.

### Steps to Reproduce

Try to log in, and get the message: "This user account is blocked."

**Comment by amyegan** [comment, marked as answer]
Hey @iamyahia. Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. To protect against platform abuse, Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Comment by noblez-90z** [comment]
I'm having same issues as stated above and i have contacted registration@vercel.com and i haven't receive any help concerning it 
please i need help
